modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Models/PublisherExtensions.cs

```diff
@@ -47,9 +50,10 @@ public static class PublisherExtensions {
                 DisplayName = model.DisplayName,
                 ExpandedNodeId = model.ExpandedNodeId,
                 HeartbeatInterval = model.HeartbeatInterval,
-                OpcPublishingInterval = model.OpcPublishingInterval,
```

Comment:
Wouldn't it be better to use `OpcPublishingInterval` since `OpcPublishingIntervalTimespan` always works with that one behind the scene ?

Reply:
fixed

Reply:
done